src/implementations/twig/components/media-container/media-container.story.js

```diff
@@ -10,11 +10,19 @@ import mediaContainer from './media-container.html.twig';
 import notes from './README.md';
 
 const getArgs = (data) => {
-  const args = {
+  let args = {
     show_description: true,
     description: data.description,
     full_width: false,
   };
+  if (data.video) {
+    args = {
+      ...args,
+      autoplay: false,
+      muted: false,
+      loop: false,
```

Comment:
The idea of COMM here is to have a single control that would toggle the different video options: autoplay, muted, loop and controls.
We should keep the parameter in the video, as you have them, but just expose one control in the media container (either we have a normal video player, or we have an autoplay video like on the banner)